ui/components/Game.jsx

```diff
@@ -39,8 +39,7 @@ export default class Game extends React.Component {
     }
 
     if (player.exitAt) {
-      const steps = exitSteps && exitSteps(game, player);
-
+      const steps = exitSteps(game, player) ? exitSteps(game, player) : [];
```

Comment:
You don't need to call `existSteps()` twice here (unnecessary computation), you can simply do an `||` instead of using a ternary operator.
`const steps = exitSteps(game, player) || [];`